client/Makefile

```diff
@@ -0,0 +1,6 @@
+SHELL := /bin/bash -ex
+
+.PHONY: format
+format:
+	black .
+	isort .
```

Comment:
Does `black` have a mode that allows us to check whether anything is in need of formatting, similar to Spotless's `spotlessCheck` target? I'm thinking of a potential future in which we may want to check formatting/linting in CI.

Reply:
Yes: https://mail.google.com/mail/u/0/?zx=85jeogl5600k#inbox/FMfcgzGpGTBZgVPrLqxvHFWPfBpGDbrT

I thought of adding a GH action. Is that what we plan on using when folks submit a PR against this repo? 

In any case, we can add that in a future PR after pylint and mypy have been added as well.

Reply:
Sorry wrong link: https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#writeback-and-reporting

Reply:
Ok sounds good. Not sure if there's any near-term intent to use Github Actions, but as you say, we can address that in a future PR.